src/miral/basic_window_manager.cpp

```diff
@@ -2981,3 +2993,28 @@ void miral::BasicWindowManager::update_application_zones_and_attached_windows()
         area->zone_policy_knows_about = area->application_zone;
     }
 }
+
+void miral::BasicWindowManager::move_cursor_to(mir::geometry::PointF point)
+{
+    action_queue->enqueue(this, [
```

Comment:
I don't think the `ServerActionQueue` is the right executor - this could end up executing on the current thread: `linearising_executor` would be better.